deb/build/debian/jenkins.init

```diff
@@ -13,91 +13,54 @@
 # Description:       Controls @@SUMMARY@@
 ### END INIT INFO
 
-PATH=/bin:/usr/bin:/sbin:/usr/sbin
-
+DAEMON=none
```

Comment:
We do our own handling of the Java command, so set this to `none` to opt out of the handling offered by `init-d-script(5)`).